extensions/cache/runtime/src/main/java/io/quarkus/cache/runtime/caffeine/CaffeineCache.java

```diff
@@ -54,6 +60,9 @@ public CaffeineCache(CaffeineCacheInfo cacheInfo, Executor executor) {
     }
 
     public Object get(Object key, Callable<Object> valueLoader, long lockTimeout) throws Exception {
+        if (key == null) {
```

Comment:
You could have used `Objects.requireNonNull` to save some typing, but that's fine :wink: